Cargo.toml

```diff
@@ -21,18 +21,18 @@ open_url = ["webbrowser"]
 default_fonts = ["egui/default_fonts"]
 
 [dependencies]
-bevy = { version = "0.8", default-features = false, features = ["bevy_render", "bevy_core_pipeline", "bevy_asset"] }
+bevy = { version = "0.9.0", default-features = false, features = ["bevy_render", "bevy_core_pipeline", "bevy_asset"] }
```

Comment:
@mvlabat 
Thanks for merging it! <3

btw; Is there a reason why the version is fixed to `0.9.0` instead of `0.9`? :)

Reply:
Hi! No particular reason. In `0.9.0` behaves the same way as `^0.9.0` (and `0.9`). If `0.9.1` is released, this won't prevent to install it.
https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#specifying-dependencies-from-cratesio

Reply:
Thanks For Letting Me Know. :3